pageserver/src/metrics.rs

```diff
@@ -558,6 +588,30 @@ pub(crate) static STORAGE_IO_TIME: Lazy<HistogramVec> = Lazy::new(|| {
     .expect("failed to define a metric")
 });
 
+#[derive(Debug)]
+struct StorageIoTime {
+    metrics: [Histogram; StorageIoOperation::COUNT],
+}
+
+impl StorageIoTime {
+    fn new() -> Self {
```

Comment:
Now the `STORAGE_IO_TIME` could be moved within this `fn new` because it should not be accessed from elsewhere, right? Probably does not need any `static` or `Lazy` either.

Reply:
yes, right. @koivunej